packaging/hipify-clang.postinst

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -e
+
+# Soft-link to bin files
+HIPBINDIR="/opt/rocm/hip/bin"
+ROCMBINDIR="/opt/rocm/bin"
```

Comment:
What's the problem in using `CPACK_PACKAGING_INSTALL_PREFIX` set in CMakeLists.txt instead?